cmd/snap/cmd_info.go

```diff
@@ -522,35 +523,42 @@ func (x *infoCmd) Execute([]string) error {
 		}
 		// stops the notes etc trying to be aligned with channels
 		w.Flush()
-		maybePrintType(w, both.Type)
-		maybePrintBase(w, both.Base, x.Verbose)
-		maybePrintID(w, both)
-		if local != nil {
-			if local.TrackingChannel != "" {
-				fmt.Fprintf(w, "tracking:\t%s\n", local.TrackingChannel)
+		maybePrintType(w, theSnap.Type)
+		maybePrintBase(w, theSnap.Base, x.Verbose)
+		if direct != nil && x.Verbose && !osutil.IsDirectory(snapName) {
+			sha3_384, _, _ := asserts.SnapFileSHA3_384(snapName)
+			if sha3_384 != "" {
+				fmt.Fprintf(w, "sha3-384:\t%s\n", sha3_384)
 			}
-			if !local.InstallDate.IsZero() {
-				fmt.Fprintf(w, "refresh-date:\t%s\n", x.fmtTime(local.InstallDate))
+		} else {
```

Comment:
this else seems wrong, shouldn't be just on direct == nil

Reply:
yeah, looks like i goofed (undetectably)

Reply:
does it mean we are missing a test?

Reply:
it's undetectable because every print inside the `else` is a maybe so nothing is printed in the cases where it's done a bit of additional work